<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>610</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora Iphone</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>calculator.ico</normaloff>calculator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-color: #000
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn7{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn8{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn9{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVezes">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnVezes{
background-color: #e69706;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn5{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnMenos{
background-color: #e69706;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn4{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn6{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn2{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMais">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnMais{
background-color: #e69706;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn1{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn3{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIgual">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnIgual{
background-color: #e69706;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>191</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn0{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVirgula">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnVirgula{
background-color: #606060;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInverter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnInverter{
background-color: #ccc;
font-size: 32px;
border-radius: 45px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivisao">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnDivisao{
background-color: #e69706;
border-radius: 45px;
font-size: 32px;
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnLimpar{
background-color: #ccc;
font-size: 32px;
border-radius: 45px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorcentagem">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPorcentagem{
background-color: #ccc;
font-size: 32px;
border-radius: 45px;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#resultado{
color: #fff;
font-size: 50px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
